snappass/templates/preview.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="container">
+  <section>
+    <div class="page-header">
+      <h1>Secret</h1>
+    </div>
+    <p class="lead">You can only open it once!</p>
+    <div class="row">
+      <div class="col-sm-6 margin-bottom-10">
+        <button id="openSecret" type="button" class="btn-lg btn-primary">Open secret</button>
```

Comment:
I think "View secret" might be clearer than "Open secret". What do you think?

Reply:
I agree that "View secret" is clearer. Would you prefer text "View secret **once**" on the button (current implementation now)? Sometimes I find that it would be helpful for the receiving part.

Reply:
Or maybe "Reveal secret".

Reply:
I think that your suggestion "Reveal secret" is a good one also. Would you like me to change?

Reply:
Yes, let's go with that. Thanks!

Reply:
And I suppose that the text should be changed to "You can only reveal ~~open~~ it once!" ?

Reply:
Agreed.

Reply:
I have updated the code now